halo2_gadgets/src/sinsemilla/merkle/chip.rs

```diff
@@ -98,13 +107,11 @@ where
         // The message pieces `a`, `b`, `c` are constrained by Sinsemilla to be
         // 250 bits, 20 bits, and 250 bits respectively.
         //
-        /*
-            The pieces and subpieces are arranged in the following configuration:
-            |  A_0  |  A_1  |  A_2  |  A_3  |  A_4  | q_decompose |
-            -------------------------------------------------------
-            |   a   |   b   |   c   |  left | right |      1      |
-            |  z1_a |  z1_b |  b_1  |  b_2  |   l   |             |
-        */
+        // The pieces and subpieces are arranged in the following configuration:
+        // |  A_0  |  A_1  |  A_2  |  A_3  |  A_4  | q_decompose |
+        // -------------------------------------------------------
+        // |   a   |   b   |   c   |  left | right |      1      |
+        // |  z1_a |  z1_b |  b_1  |  b_2  |   l   |      0      |
```

Comment:
`q_decompose` is zero here, but blank in a later comment. I think it should be blank (it is a simple selector anyway so it might get compressed with another selector)